README.md

```diff
@@ -65,7 +65,7 @@ We are expecting to see how many fruits are available today after the work shift
 ```
 The line above means you have 152 bananas, and 90 apples in your Fruit store after the work shift.
 
-**Hint: Think about creating some FruitTransaction model to store info from file line for more convenient data processing 
+**Hint: Think about creating some FruitTransaction core.basesyntax.db.model to store info from file line for more convenient data processing 
```

Comment:
do not change README.md file

Reply:
not fixed
be careful with project-wide replacements! And always review the files to be committed

Reply:
Return it back